AK/SinglyLinkedList.h

```diff
@@ -81,9 +81,15 @@ class SinglyLinkedList {
 public:
     SinglyLinkedList() = default;
     SinglyLinkedList(const SinglyLinkedList& other) = delete;
-    SinglyLinkedList(SinglyLinkedList&&) = default;
+    SinglyLinkedList(SinglyLinkedList&& other)
+        : m_head { move(other.m_head) }
+        , m_tail { move(other.m_tail) }
```

Comment:
`move()` does not do anything here, as `m_head` and `m_tail` are both primitive raw pointer values.